setup.cfg

```diff
@@ -19,7 +19,7 @@ replace =
 	unreleased
 	==========
 	
-	{new_version} {utcnow:%%Y-%%m-%%d}
+	({new_version} {utcnow:%%Y-%%m-%%d})
```

Comment:
The brackets just need to wrap the date like;

`{new_version} ({utcnow:%%Y-%%m-%%d})`